data/features.js

```diff
@@ -190,7 +190,7 @@ export const ourServices = [
   },
   {
     iconSrc: "/assets/img/icon/feature-icon1-3.svg",
-    title: "UI/UX Design",
```

Comment:
Why is this removed?

Reply:
Using `_` for `/` and `-` for ` ` (spaces) to encode and decode properly.